src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -74,6 +77,9 @@ public static void Initialize()
         {
             if (TracerSettings.LoadTracerAtStartup)
             {
+                // Initialize SdkSelfDiagnosticsEventListener to create an EventListener for the OpenTelemetry SDK
+                _sdkEventListener = new(EventLevel.Warning);
```

Comment:
Perhaps we should make `IsDebugEnabled()` accessible here so if it is on we use a more verbose level.

Reply:
We can do this in a follow up PR.